settings/trex.toml

```diff
@@ -13,6 +13,11 @@ predefined = "web_mercator"
 
 [[tileset]]
 name = "vapour_trail"
+attribution = "<a href='http://junglebus.io/'>Jungle Bus</a> <a href='http://www.openstreetmap.org/copyright'>© OpenStreetMap contributors</a>"
+minzoom = 6
+#maxzoom = 22
+#center = 0, 0 # longitude, latitude
+start_zoom = 10
```

Comment:
Thanks for the attribution ! 
I've seen in t-rex that the minzoom and maxzoom are computed if not provided. Why to you want to set the minzoom ? (default value will be 6).
Can you help me understand what the start_zoom is used for ? It's a start value for the MapBox client map ?

Reply:
You are right, the meaning have changed since my initial PR for this.

Reply:
Thanks ! What about the start_zoom ? Sorry to bother you with it, but I don't understand...

Reply:
Yes, all this param are for MapBox client